packages/fleet/lib/supervisor/supervisor.ts

```diff
@@ -428,17 +428,14 @@ export class Supervisor {
     }
 
     private async finishingTimeout({ node }: { type: 'FINISHING_TIMEOUT'; node: Node }): Promise<Result<Node>> {
-        // Locally we assume the node is IDLE
-        // since the process is likely already killed
-        // and the node is not able to notify back that it is idle
-        if (envs.RUNNER_TYPE === 'LOCAL') {
-            return nodes.transitionTo(this.dbClient.db, {
-                nodeId: node.id,
-                newState: 'IDLE'
-            });
-        }
-        // TODO: find a better way to warn and alert
-        logger.warning('Node is taking too long to finish:', node);
```

Comment:
I basically didn't think this state was possible. I evidently forgot that a runner can crash while being in finishing state and never report about being idle 🥲